.env

```diff
@@ -5,7 +5,16 @@ SIMORGH_PUBLIC_STATIC_ASSETS_PATH=/
 SIMORGH_ATI_BASE_URL=https://logws1363.ati-host.net?
 SIMORGH_CONFIG_URL=https://config.test.api.bbci.co.uk/
 SIMORGH_INCLUDES_BASE_URL=https://www.test.bbc.com/ws/includes
+SIMORGH_INCLUDES_BASE_AMP_URL=https://news.test.files.bbci.co.uk
```

Comment:
Oops (think this file is in the .gitignore list?)

Reply:
I don't know what this file is for and why it isn't gitignored; I've reverted the change? I keep meaning to bring this up, we only use the env configs here to my knowledge: https://github.com/bbc/simorgh/tree/latest/envConfig

Reply:
Not 100% sure, I think it's to standardise the start up scripts for all environments - the relevant file is always copied to .env on build? But for some reason the .env file lives in version control 🤷‍♀️ (it's supposed to look like this https://github.com/bbc/simorgh/commit/3fb7b2077777229cfb722c97ee6fdcf7ef730bf6#diff-f579cccc964135c7d644c7b2d3b0d3ec)